src/components/Topics.jsx

```diff
@@ -26,26 +22,27 @@ export default function Topics(props){
     }, [])
 
     function handleClick(topicName) {
-        setCurrentTopic(topicName)
-        setArticles(()=>{
-            if(topicName==='all articles'){
-                return allArticles
-            }
-            return allArticles.filter((article)=>{
-                return article.topic === topicName
-            })
-        })
+        // setIsLoadingArticles(true)
+
+        // getArticles(topicName)
+        // .then((articleData)=>{
+        //     setIsLoadingArticles(false)
+        //     setArticles(articleData)
+        // })
         }
 
     return (
         <>
     <section id = "topics">
-        {topics.map((topic)=>{
-            return <TopicCard key = {topic.slug} topic = {topic} handleClick={handleClick}/>
-        })}
+            {isLoading? <p>Loading topics...</p> : (
```

Comment:
Great that you've implemented a loading pattern and you're loading the topics using an API request rather than hard coding these values